roles/confluent.common/tasks/ubuntu.yml

```diff
@@ -33,6 +34,7 @@
   file:
     path: /usr/share/man/man1
     state: directory
+    mode: 0755
```

Comment:
looks like this is how it is by default on systems
```
[root@kafka-broker1 /]# ls -lat /usr/share/man
total 100
drwxr-xr-x 1 root root 4096 Feb  5 17:07 ..
drwxr-xr-x 1 root root 4096 Feb  5 17:07 man1
```
want to make sure this does not change things

Reply:
But that looks like `0755`, doesn't it? Or?

Reply:
yes i was just saying "this looks good, i double checked"